.changeset/serious-bees-burn.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: false negatives for `value-no-vendor-prefix`
```

Comment:
```suggestion
Fixed: `value-no-vendor-prefix` false negatives for more vendor prefixes
```

Reply:
> for more

That sounds weird. Ill try to reformulate so that the rule is first.